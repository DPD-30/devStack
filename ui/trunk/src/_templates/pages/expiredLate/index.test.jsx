import React from 'react'
import { renderToString } from 'react-dom/server'


import TestComponent from './index.jsx'

jest.unmock('./index.jsx')


test('Expired and Late report matches snapshot', () => {
	const comp = renderToString(<TestComponent />)

	expect(comp).toMatchSnapshot()
})
