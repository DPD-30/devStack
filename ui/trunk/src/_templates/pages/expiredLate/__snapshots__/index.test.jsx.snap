// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expired and Late report matches snapshot 1`] = `
<div data-reactroot>
  <h2>
    <div>
      mock icon
    </div>
    Expired
  </h2>
  <table class="table table-striped">
    <thead>
      <tr>
        <th>
          Application Name
        </th>
        <th style="text-align:center">
          Status
        </th>
        <th>
          Owner
        </th>
        <th style="text-align:center">
          IMatrix #
        </th>
        <th style="text-align:center">
          Phase
        </th>
        <th>
          POC
        </th>
        <th style="text-align:center">
          Expiration
        </th>
        <th style="text-align:center">
          Icon
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-09-20
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          AWRE
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-09-20
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          Pagasys
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-09-20
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          NBS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-05-20
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          MdM
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-09-23
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          AWARE
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-09-13
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          NBS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          2
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-05-20
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
    </tbody>
  </table>
  <br>
  <h2>
    <div>
      mock icon
    </div>
    Upcoming Expirations
    <small>
      (within 60 days)
    </small>
  </h2>
  <table class="table table-striped">
    <thead>
      <tr>
        <th>
          Application Name
        </th>
        <th style="text-align:center">
          Status
        </th>
        <th>
          Owner
        </th>
        <th style="text-align:center">
          IMatrix #
        </th>
        <th style="text-align:center">
          Phase
        </th>
        <th>
          POC
        </th>
        <th style="text-align:center">
          Expiration
        </th>
        <th style="text-align:center">
          Icon
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          MdM
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          6
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-10-23
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
      <tr>
        <td>
          AWARE
        </td>
        <td style="text-align:center">
          1
        </td>
        <td>
          IMS
        </td>
        <td style="text-align:center">
          1
        </td>
        <td style="text-align:center">
          3
        </td>
        <td>
          User1
        </td>
        <td style="text-align:center">
          2020-10-13
        </td>
        <td style="text-align:center">
          <div>
            mock icon
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
